unsteady/solver.py

```diff
@@ -1451,7 +1475,10 @@ def solve_forward_step(self, i, update_forcings=None, export_func=None, restarte
 
             # Mesh movement
             if self.iteration % op.dt_per_mesh_movement == 0:
-                inverted = self.move_mesh(i)
+                if self.iteration == 0 and op.reinitialise:
+                    inverted = self.move_mesh(i, reinit=True)
+                else:
+                    inverted = self.move_mesh(i)
```

Comment:
Could put
`inverted = self.move_mesh(i, reinit=self.iteration == 0 and op.reinitialise)`

Reply:
changed